### ISSUE TITLE
Sessions room

### ISSUE THREAD

**Issue by vuillaut** [vgvassilev/hepcon-config#29]
Hi
Unless I missed it I cannot see in what room each specific session is held (e.g. in the session panel).
Would be great to have this info. Thanks.

**Comment by vgvassilev** [comment]
I agree, we should it make more visible. Currently, you can get that information if you go to 'sessions', select the session you are interested in and click on a talk. There you will see the room number. Let me know if that was not what you meant.

**Comment by mvassilev** [comment]
Closing as this is fixed.
